### DISCUSSION TITLE
[4주차] [온라인 스터디] 글 모음

### DISCUSSION THREAD

**Discussion by bigsaigon333** [fe-backpackers/fe-interview-basecamp#204] [General]
## 순서

~22:05 체크인

~22:20  입풀기(with 카훗)

~22:35 다른 PR 참조하여 내 PR 수정하기. 궁금한 점, 모르는 점은 PR에 질문달기

~23:00 3인 모의면접. 1:1로 진행하며, 남은 1명은 관찰자.

**Comment by bigsaigon333** [comment]
## 체크인(~22:05)

오늘의 기분을 10점 만점으로 이유와 함께 기재해주세요!

**Reply by bigsaigon333** [reply]
7점: 지원한 회사의 1차 면접이 끝나서 후련하네요! 결과는 2주 뒤에 나오지만.. 그동안 알차게 보낼 생각입니다~!

**Reply by jinsoobyeon** [reply]
10점 - 스터디해서 기분 좋습니다.

**Reply by Lobo2u** [reply]
7점-피곤하네요 ㅠㅠ

**Reply by MINGDY98** [reply]
8점 : 저도 면접 결과가 일주일뒤에나와서 피가 마르는 중입니다..ㅜㅜ

**Reply by ChaerinYu** [reply]
7점: 늦었습니다.. 피곤하네요ㅠㅠ

**Comment by bigsaigon333** [comment]
## 입풀기 with 카훗 (~22:20)

<https://kahoot.it>

PIN:

**Comment by bigsaigon333** [comment]
## 스터디 설문 

4주간의 인터뷰 베이스캠프 스터디 후 간단한 설문을 통해 앞으로의 스터디 방향을 점검하고자 합니다.

<https://forms.gle/KESYwGRj2H5dpuBW7>
